### PR TITLE
[8.0][ADD] report_xlsx_helper

### PR DESCRIPTION
**Pull request by luc-demeyer**
backport of 10.0 report_xlsx_helper module

### CODE REVIEW

**Comment by luc-demeyer** [comment]
@astirpe : can you review (I fixed a couple of flake8/pylint issues in your modules to greenify this PR).

File: report_xlsx_helper/README.rst [hunk 1]
```diff
@@ -0,0 +1,89 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+===========================
+Excel report engine helpers
```
Comment (pankk) [inline report_xlsx_helper/README.rst:6]: Rename helpers to helper

File: report_xlsx_helper/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Copyright 2009-2018 Noviat.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Report xlsx helpers',
```
Comment (pankk) [inline report_xlsx_helper/__openerp__.py:6]: Rename helpers to helper

**Review by pankk — CHANGES_REQUESTED** [review]

**Review by pankk — APPROVED** [review]
Successful functional test with six custom xlsx reports using this helper.

File: report_xlsx_helper/__openerp__.py (hunk 2 above)
Comment (luc-demeyer) [reply report_xlsx_helper/__openerp__.py:6]: This module add a set tools to facilitate the creation of XLSX files, hence I think 'helpers' in the README is more appropriate.

Comment (pankk) [reply report_xlsx_helper/__openerp__.py:6]: Ok, I didn't think of it that way. I began to think of it as a "helper module". I think usually such a module would be named in plural form anyway, but the singular convention is a good measure for headache avoidance. But I now agree that the description should use the plural form to be accurate.

**Pull request closed without merging**